web/src/stream_popover.js

```diff
@@ -214,7 +214,29 @@ function build_stream_popover(opts) {
     });
 }
 
-export function build_move_topic_to_stream_popover(
+async function get_message_type(current_stream_id, topic_name, current_message_id) {
```

Comment:
This probably needs a clearer name, like `get_message_position_in_conversation`

Reply:
`get_message_type` has been renamed to `get_message_placement_in_conversation` in the updated commit.